### ISSUE TITLE
code block wrapping

### ISSUE THREAD

**Issue by kpurdon** [DandyDev/pelican-bootstrap3#207]
Ideally you don't want to wrap code-block text, instead preferring horizontal scroll since formatting matters.

I solved this with CUSTOM_CSS

```css
pre {
  overflow: auto;
  word-wrap: normal;
  white-space: pre;
}
```

Perhaps this (or another solution) could be activated with a new option `CODE_OVERFLOW` or something like that.

**Comment by smetj** [comment]
+1

**Comment by DonDebonair** [comment]
Hi!

First of all, thanks for your interest in this theme! It's with heavy heart that I have decided that I can no longer maintain this theme due to lack of time and changing interests. To makes sure the theme lives on however and not goes to waste, I've donated it to the [Pelican community](https://github.com/getpelican/pelican-themes). It can now be found in the official [pelican-themes repository](https://github.com/getpelican/pelican-themes/tree/master/pelican-bootstrap3). I'd suggest you re-submit your issue  there.

Sorry for the inconvenience!
